packages/terra-progress-bar/docs/README.md

```diff
@@ -15,29 +15,29 @@ Note: The progress bar displays fill with respect to percentage (value between 0
 ```jsx
 import React from 'react';
 import ProgressBar from 'terra-progress-bar';
+import './colors.css';
```

Comment:
I feel like we should remove this. We have the code examples people can toggle

Reply:
Fixed in https://github.com/cerner/terra-core/pull/2211/commits/c258cbe340b99ed9dfed1b895fb1e76bba2b265a